### PR TITLE
fix topo click node display redundant client and server element

### PR DESCRIPTION
**Pull request by x22x22**
fix topo click node display redundant client and server element

### CODE REVIEW

File: src/graph/query/topology.ts [hunk 1]
```diff
@@ -53,7 +53,9 @@ export const queryTopoInstanceClientInfo =
...
 
 export const queryDependencyInstanceServerMetric =
-  `query queryDependencyInstanceServerMetric(${DependencyInstanceServerMetric.variable}) {${DependencyInstanceServerMetric.query}}`;
+  `query queryDependencyInstanceServerMetric(${DependencyInstanceServerMetric.variable}) \
+    {${DependencyInstanceServerMetric.query}}`;
 
 export const queryDependencyInstanceClientMetric =
-  `query queryDependencyInstanceClientMetric(${DependencyInstanceClientMetric.variable}) {${DependencyInstanceClientMetric.query}}`;
+  `query queryDependencyInstanceClientMetric(${DependencyInstanceClientMetric.variable}) \
+    {${DependencyInstanceClientMetric.query}}`;
```
Comment (x22x22) [inline src/graph/query/topology.ts]: wipe off "length greater than 120" warn.

File: src/views/components/topology/topo-detect-point.vue [hunk 2]
```diff
@@ -17,117 +17,121 @@
...
+            class="link-topo-aside-box-btn tc r sm cp b"
+            @click="setMode(false)"
+        >{{ this.$t('client') }}</span>
+        <span
+            v-if="stateTopo.detectPoints.indexOf('SERVER') !== -1"
+            :class="{'active':stateTopo.mode}"
+            class="link-topo-aside-box-btn tc r sm cp b"
+            @click="setMode(true)"
+        >{{ this.$t('server') }}</span>
+      </div>
```
Comment (x22x22) [inline src/views/components/topology/topo-detect-point.vue]: Only modified here, others are formatting changes.

File: src/views/components/topology/topo-detect-point.vue [hunk 3]
```diff
@@ -210,7 +214,7 @@
       this.GET_INSTANCE_DEPENDENCY({
         serverServiceId: this.stateTopo.selectedServiceCall.source.id,
         clientServiceId: this.stateTopo.selectedServiceCall.target.id,
-        duration: this.durationTime,
+        duration: this.durationTime
```
Comment (Fine0830) [inline src/views/components/topology/topo-detect-point.vue]: Missing trailing comma

File: src/views/components/topology/topo-detect-point.vue [hunk 4]
```diff
@@ -171,7 +175,7 @@
           .then(() => {
             this.GET_QUERY({
               serviceId: service.id || '',
-              duration: this.durationTime,
+              duration: this.durationTime
```
Comment (Fine0830) [inline src/views/components/topology/topo-detect-point.vue]: Missing trailing comma

File: src/views/components/topology/topo-detect-point.vue [hunk 5]
```diff
@@ -17,117 +17,121 @@
...
-  import TopoInstanceDependency from './topo-instance-dependency.vue';
   import TopoChart from './topo-chart.vue';
+  import TopoInstanceDependency from './topo-instance-dependency.vue';
   import ChartResponse from './topo-response.vue';
 
   @Component({
     components: {
-      TopoInstanceDependency, ChartResponse, TopoChart,
-    },
+      TopoInstanceDependency, ChartResponse, TopoChart
```
Comment (Fine0830) [inline src/views/components/topology/topo-detect-point.vue]: Missing trailing comma

**Comment by wu-sheng** [comment]
@Fine0830 These should be fixed by #237

**Pull request merged**